src/etcher.rs

```diff
@@ -16,6 +15,12 @@ use crate::timer::Timer;
 pub fn rip_bytes(path: &str) -> anyhow::Result<Vec<u8>> {
     let byte_data = fs::read(path)?;
 
+    if byte_data.len() == 0 {
+        eprintln!(
+            "Empty files cannot be embedded! File names are not retained, so its pointless anyway"
```

Comment:
```suggestion
            "Empty files cannot be embedded! File names are not retained, so it's pointless anyway"
```
Typo :slightly_smiling_face: